src/domain/transfer/index.js

```diff
@@ -33,6 +33,7 @@ const StreamingProtocol = require('@mojaloop/central-services-shared').Util.Stre
 const ErrorHandler = require('@mojaloop/central-services-error-handling')
 const Config = require('../../lib/config')
 const generalEnum = require('@mojaloop/central-services-shared').Enum
+const safeStringify = require('fast-safe-stringify')
```

Comment:
Why this?

Reply:
Circular references in the below `topicConfig` and `kafkaConfig`